lib/nolibc/include/stdio.h

```diff
@@ -93,6 +98,13 @@ int puts(const char *s);
 
 #if CONFIG_LIBVFSCORE
 int rename(const char *oldpath, const char *newpath);
+int feof(struct vfscore_file *stream);
+int fseek(struct vfscore_file *stream, long offset, int whence);
+int fclose(struct vfscore_file *stream);
+struct vfscore_file *fopen(const char *pathname, const char *mode);
```

Comment:
I am a bit confused, probably because of my lack of understanding. Won't this definition be in conflict with this one `typedef struct _nolibc_fd FILE` from line 54?